Infrastructure/envSetup.ps1

```diff
@@ -3,12 +3,11 @@ New-Module -Name EnvSetup -ScriptBlock {
     $sourceNugetExe = "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe"
     $targetNugetExe = ".\nuget.exe"
     Invoke-WebRequest $sourceNugetExe -OutFile $targetNugetExe
-    
-    echo $env:DotFeedUri
-    
-    Set-Alias nuget $targetNugetExe -Scope Global -Verbose
+    Set-Alias nuget $targetNugetExe -Verbose
     
     # Add DotFeed as source
-    nuget sources add -Name DotFeed -Source $env:DotFeedUri -username buildAgent -password $env:DotFeedPassword
-    nuget sources list
+    Write-Host (nuget sources add -Name DotFeed -Source $env:DotFeedUri -username buildAgent -password $env:DotFeedPassword)
```

Comment:
It seems a little strange that the actual command is contained inside of the `Write-Host`. Is there a way to pipe the output to the console?